File: Mixter/Domain/Core/Messages/Events/ReplyMessagePublished.cs
using Mixter.Domain.Identity;

namespace Mixter.Domain.Core.Messages.Events
{
    public struct ReplyMessagePublished : IDomainEvent
    {
        public MessageId ReplyId { get; private set; }

        public UserId Replier { get; private set; }

        public string ReplyContent { get; private set; }
        
        public MessageId ParentId { get; private set; }

        public ReplyMessagePublished(
            MessageId replyId, 
            UserId replier, 
            string replyContent, 
            MessageId parentId)
            : this()
        {
            ReplyId = replyId;
            Replier = replier;
            ReplyContent = replyContent;
            ParentId = parentId;
        }

        public object GetAggregateId()
        {
            return ReplyId;
        }
    }
}

File: app/Domain/Messages/ReplyMessagePublished.php
<?php

namespace App\Domain\Messages;

use App\Domain\Identity\UserId;
use App\Domain\IDomainEvent;

class ReplyMessagePublished implements IDomainEvent
{
    /**
     * @var MessageId
     */
    private $replyId;

    /**
     * @var string
     */
    private $replyContent;

    /**
     * @var UserId
     */
    private $replierId;

    /**
     * @var MessageId
     */
    private $parentMessageId;

    public function __construct(MessageId $replyId, $replyContent, UserId $replierId, MessageId $parentMessageId)
    {
        $this->replyId = $replyId;
        $this->replyContent = $replyContent;
        $this->replierId = $replierId;
        $this->parentMessageId = $parentMessageId;
    }

    /**
     * @return MessageId
     */
    public function getReplyId()
    {
        return $this->replyId;
    }

    /**
     * @return string
     */
    public function getReplyContent()
    {
        return $this->replyContent;
    }

    /**
     * @return UserId
     */
    public function getReplierId()
    {
        return $this->replierId;
    }

    /**
     * @return MessageId
     */
    public function getParentMessageId()
    {
        return $this->parentMessageId;
    }

    /**
     * @return string
     */
    public function getAggregateId()
    {
        return $this->getReplyId();
    }
}